systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaBridgeResource.java

```diff
@@ -78,6 +79,7 @@ public static KafkaBridge kafkaBridgeWithoutWait(KafkaBridge kafkaBridge) {
 
     public static void deleteKafkaBridgeWithoutWait(KafkaBridge kafkaBridge) {
         kafkaBridgeClient().inNamespace(ResourceManager.kubeClient().getNamespace()).delete(kafkaBridge);
+        ResourceManager.waitForResourceDeletion(kafkaBridge, KafkaBridgeResources.deploymentName(kafkaBridge.getMetadata().getName()));
```

Comment:
again this is not deleteKafkaBridgeWithoutWait. Here you are waiting on the bridge resource to be deleted or am i reading it wrongly? @Frawless

Reply:
same as above